mentat/code_file_manager.py

```diff
@@ -31,13 +31,11 @@ def __init__(self):
 
     def read_file(self, path: Path) -> list[str]:
         session_context = SESSION_CONTEXT.get()
-        git_root = session_context.git_root
 
-        abs_path = path if path.is_absolute() else Path(git_root / path)
-        rel_path = Path(os.path.relpath(abs_path, git_root))
+        abs_path = path if path.is_absolute() else session_context.cwd.joinpath(path)
```

Comment:
Can we use the / operator instead of joinpath? Also, we don't actually need to check for is_absolute; if you do the / operator and the second argument is absolute already it won't change it at all (although for clarity's sake I'm perfectly fine with keeping the is absolute check)

Reply:
> Can we use the / operator instead of joinpath?

Sure

>if you do the / operator and the second argument is absolute already it won't change it at all

Huh I didn't know that was a python "feature" haha. I vote for keeping the explicit check in.